docs/contributing/translation/sync-guide.md

```diff
@@ -4,6 +4,12 @@ title: Keeping Translations Up-to-date
 
 Periodically, gatsbybot will update your translation repo to be up-to-date with the current English repo. If there is an update to a page that is already translated, gatsbybot will create a pull request listing the conflicts between translation and the new English content. Resolving these conflicts and merging these pull requests is essential to keeping your translation repo up-to-date.
 
+## Gatsbybot sync behavior
+
+The gatsbybot will run the sync script on every translation repo on Friday at 6:00 PM PST. It will create up to two pull requests: one for changes that do not have conflicts, which can me immediately merged, and one for changes that do have conflicts, which will require updates and code review.
+
+If there are existing sync pull requests marked by the `sync` label, gatsbybot will skip the repository until the next time it runs. Because of this, it is important to make sure that you merge sync and conflict pull requests promptly, so that changes stale translations don't build up.
```

Comment:
```suggestion
If there are existing sync pull requests marked by the `sync` label, gatsbybot will skip the repository until the next time it runs. Because of this, it is important to make sure that you merge sync and conflict pull requests promptly, so that changes stale translations don't build up.
```

I think there's a word missing in the last clause but I'm not sure what it is. Could you clarify what you were going for there?